care/abdm/utils/fhir.py

```diff
@@ -88,7 +88,10 @@ def _practioner(self):
 
         id = str(uuid())
         name = (
-            self.consultation.verified_by
+            (
+                self.consultation.verified_by
+                and f"{self.consultation.verified_by.first_name} {self.consultation.verified_by.last_name}"
```

Comment:
are we not using fhir for old records?
cc @khavinshankar

Reply:
Yes, we will be generating fhir records for older consultations too, so we might need a fallback to the legacy field